method/group.js

```diff
@@ -126,18 +126,44 @@ module.exports = function GroupMethods (ssb, keystore, state) {
       }]
 
       pull(
-        ssb.query.read({ query }),
+        ssb.query.read({ query: addedQuery }),
         pull.filter(addMemberSpec.isValid),
         pull.filter(msg =>
           groupId === msg.value.content.recps[0]
         ),
         pull.map(msg => msg.value.content.recps.slice(1)),
         pull.flatten(),
         pull.unique(),
-        pull.collect((err, members) => {
+        pull.collect((err, addedMembers) => {
```

Comment:
At first I was hoping to do some fancy pull stream stuff to get the streams to run in parallel, but after staring at the docs for a bit I decided that it's better to get to work simply to start with.